src/helpers/ui/sheet_interactive.ts

```diff
@@ -38,3 +38,23 @@ export function interactiveRenameSheet(env: SpreadsheetChildEnv, sheetId: UID, e
     error: errorText,
   });
 }
+
+export function interactiveRemoveColumnsRows(
```

Comment:
Remove the interactive method, sicne we decided to go with hidden menu items instead.

Reply:
The topbar also uses the menu item, so the option won't show either :)

Reply:
Well, In my case the option still appears from the top bar edit.
Maybe my fix was not in the right place, let me check

Reply:
Mh yeah we didn't use the exact same menu item in 16.0. Extract the isVisible in a helper `CAN_REMOVE_COLUMNS` then, and put it also in the topbar edit menu item.

Reply:
I have implemented the suggested changes; It should look better now :)